mappings/net/minecraft/entity/ai/brain/ScheduleBuilder.mapping

```diff
@@ -0,0 +1,11 @@
+CLASS avz net/minecraft/entity/ai/brain/ScheduleBuilder
+	CLASS avz$a ActivityData
```

Comment:
iirc this is referred to as `ActivityEntry` from a string somewhere

Reply:
Can't find that string, but I'll rename it to `ActivityEntry` anyway.